### DISCUSSION TITLE
Blink detection with WebGazer

### DISCUSSION THREAD

**Discussion by miguelo93** [jspsych/jsPsych#1624] [Q&A]
Hi all,

I was  wondering if someone ever implemented a way to detect eye blink (eye open/closed) in jsPsych, using the WebGazer extension.

Thanks!

**Comment by derekalbert** [comment]
This would be nice. So far, I have not seen this functionality in webgazer. But, you could record video through jspsych and calculate blinks using other software's, like https://github.com/masanbasa3k/Blink-Counter
